deps/chakrashim/src/jsrtcontextshim.cc

```diff
@@ -368,11 +368,34 @@ bool ContextShim::InitializeProxyOfGlobal() {
   });
 }
 
-bool ContextShim::EnsureInitialized() {
+
+bool ContextShim::InitializeCurrentContextShim() {
```

Comment:
A bit confusing to have 2 very similar methods. How about adding a bool parameter like "bool isCurrentContext = false" to EnsureInitialized, combining with this one?